docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:latest
```

Comment:
`ubuntu:latest` points to the latest LTS release (16.04 right now), so it's not actually the very latest release.  I wanted to avoid forgetting about it later on and having an old version, but I guess it's worth the trouble to avoid a possible break when we're not expecting it.